internal/pkg/cli/local_run.go

```diff
@@ -0,0 +1,127 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/aws/copilot-cli/internal/pkg/aws/identity"
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
+	"github.com/aws/copilot-cli/internal/pkg/term/selector"
+	"github.com/aws/copilot-cli/internal/pkg/workspace"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type localRunVars struct {
+	name    string
+	appName string
+	envName string
+}
+
+type localRunOpts struct {
+	localRunVars
+
+	store store
```

Comment:
It doesn't seem this is used 🤔

Reply:
We are sending it to the selector as a parameter right? And also now we are using for Validation.

Reply:
> We are sending it to the selector as a parameter right

We use `store` the variable not `o.store`.

> And also now we are using for Validation.

This is true.